hazelcast/src/main/java/com/hazelcast/internal/management/ManagementCenterService.java

```diff
@@ -57,6 +57,7 @@
         private final LongSupplier clock;
         private volatile long mostRecentAccessTimestamp;
         private final ConcurrentMap<Address, Long> lastAccessTimestamps = new ConcurrentHashMap<>();
+        private final ConcurrentMap<Address, Integer> eventsReceivedInSameMillisec = new ConcurrentHashMap<>();
```

Comment:
Probably, this map has to be clean up under the same conditions as `lastAccessTimestamps`. Otherwise, we might have a mem leak.